### PR TITLE
fix ip rule goto bug

### PR DESCRIPTION
**Pull request by yyd01245**
ip rule goto id action type error!

### CODE REVIEW

File: rule_linux.go [hunk 1]
```diff
@@ -144,10 +144,12 @@ func ruleHandle(rule *Rule, req *nl.NetlinkRequest) error {
 		req.AddData(nl.NewRtAttr(nl.FRA_OIFNAME, []byte(rule.OifName)))
 	}
 	if rule.Goto >= 0 {
-		msg.Type = nl.FR_ACT_NOP
+		// msg.Type = nl.FR_ACT_NOP
```
Comment (aboch) [inline rule_linux.go]: Please remove the commented lines

File: rule_linux.go [hunk 2]
```diff
@@ -144,10 +144,12 @@ func ruleHandle(rule *Rule, req *nl.NetlinkRequest) error {
...
+		// fix goto bug
+		msg.Type = nl.FR_ACT_GOTO
 		b := make([]byte, 4)
 		native.PutUint32(b, uint32(rule.Goto))
-		req.AddData(nl.NewRtAttr(nl.FRA_GOTO, b))
+		req.AddData(nl.NewRtAttr(nl.FRA_GOTO, b))
```
Comment (aboch) [inline rule_linux.go]: please remove the blank space

File: rule_linux.go (hunk 1 above)
Comment (yyd01245) [reply rule_linux.go]: OK!

**Comment by aboch** [comment]
LGTM

**Pull request merged**